modules/backend/classes/BackendController.php

```diff
@@ -210,7 +218,7 @@ protected function getRequestedController($url)
          * Look for a Plugin controller
          */
         if (count($params) >= 2) {
-            list($author, $plugin) = $params;
+            [$author, $plugin] = $params;
```

Comment:
Has list() been removed or is this just a coding style preference?

Reply:
This is just coding style applied by IDE, sorry

Reply:
This is just coding style applied by IDE, sorry